README.md

```diff
@@ -42,7 +52,36 @@ $ manticore ./path/to/binary ++ ++  # use two symbolic strings of length two as
 
 ### API
 
-Manticore has a Python programming interface which can be used to implement custom analyses.
+Manticore has a Python programming interface which can be used to implement custom analyses and is capable of detailed verification of arbitrary properties of smart contracts via its Python API. Set starting conditions, identify symbolic transactions, then review discovered states to ensure invariants for your contract hold.
+
+```python
+from manticore.ethereum import ManticoreEVM
```

Comment:
i also think we should change the contract we use here. i think the contract here should, as concisely as possible, showcase the features to

- execute contract functions with symbolic input
- execute arbitrary sequences of contract functions
- peruse the discovered states, looking for states that match a certain criteria (e.g. had an exception)

I think a good candidate for this would be a contract based on the umd example, but slightly simplified, as this example is a bit long. what i like about this example is that it's not immediately obvious from a human perspective how to cause the assertion to fail, but it's trivial to use manticore to find out the answer. i think we should come up with a slightly similar example in the same vein and use it here. https://github.com/trailofbits/manticore/blob/master/examples/evm/umd_example.sol

just putting these notes here; this is pretty low priority work and we should probably focus on other things for the moment

Reply:
I don't like the UMD example because it's too "academic." I don't want a demo of features, I want to see how Manticore helps me solve a problem or achieve a goal.